hack/run-locally.sh

```
@@ -0,0 +1,121 @@
+#!/bin/bash 
+
+if [ -z "$CLUSTER_DIR" ]; then
+    echo CLUSTER_DIR is required
+    exit 1
+fi
+
+set -o errexit
+set -o nounset
+
+# Install our overrides so the cluster doesn't run the network operator.
+function override() {
+    local cluster_dir=$1
+    
+    if [ ! -e $cluster_dir/manifests/cvo-overrides.yaml ]; then
+        echo "cannot find cvo-overrides.yaml; please run"
+        echo "openshift-install --dir=$cluster_dir create manifests"
+        exit 1
+    fi
+
+    # Patch the CVO to not create the network operator
+    echo "Applying overrides to $CLUSTER_DIR/manifests/cvo-overrides.yaml"
+
+    kubectl patch --type=json --local=true -f=$CLUSTER_DIR/manifests/cvo-overrides.yaml -p "$(cat hack/overrides-patch.yaml)" -o yaml > $CLUSTER_DIR/manifests/.cvo-overrides.yaml.new
+
+    mv $CLUSTER_DIR/manifests/.cvo-overrides.yaml.new $CLUSTER_DIR/manifests/cvo-overrides.yaml
+}
+
+# Extract the image references.
+function build_env() {
+    echo 
+    echo "Pulling the latest release definition"
+    # get the images
+    RELEASE_IMAGE=${RELEASE_IMAGE:-registry.svc.ci.openshift.org/openshift/origin-release:v4.0}
+    echo "Pulling the release image $RELEASE_IMAGE"
+    echo "If this fails, you'll need to log in."
+    podman pull $RELEASE_IMAGE
+
+    echo > env.sh
+
+    echo
+    echo "Extracting image references to env.sh"
+
+    # If I were cleverer, I'd pull this from the release playload.
+    echo "export NODE_IMAGE=$(podman run --rm $RELEASE_IMAGE image node)" >> env.sh
```

I get an error printed for each of these commands:

    ERRO[0001] could not find slirp4netns, the network namespace won't be configured: exec: "slirp4netns": executable file not found in $PATH 

The script works, but the errors make it seem like something's wrong. Adding `--net none` to the `podman run` command suppresses the error